srcpkgs/python3-pylast/template

```diff
@@ -1,14 +1,15 @@
 # Template file for 'python3-pylast'
 pkgname=python3-pylast
-version=4.1.0
-revision=6
-build_style=python3-module
-hostmakedepends="python3-setuptools_scm"
-depends="python3-setuptools"
+version=5.3.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="hatchling hatch-vcs"
+depends="python3-httpx"
+checkdepends="python3-pytest python3-flaky ${depends}"
 short_desc="Python3 interface to last.fm and libre.fm"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="Apache-2.0"
 homepage="https://github.com/pylast/pylast"
 distfiles="${PYPI_SITE}/p/pylast/pylast-${version}.tar.gz"
-checksum=ad084aec1bf7e307bc42d7cc1a003851f5bee1ad24fb697a9fdc300bbfe63932
-make_check=no # The 'pylast' distribution was not found and is required by the application
+checksum=637943b1b0e6045dd85ed7389db6071a1fea45cc7ff90dc6126fd509ca6fae2f
+make_check=no # Unable to determine tests for PEP517 Python templates
```

Comment:
Most tests are skipped because pylast credentials are not available, but a few unicode tests run just fine.
```suggestion
```

Reply:
Looks like xbps is smart enough to detect pytest automatically. I hope this would just suffice.

<details><summary>Test log</summary>
<p>

```sh
$ ./xbps-src check ${PKG_CHK}
=> xbps-src: updating repositories for host (x86_64)...                                                                       
[*] Updating repository `https://repo-default.voidlinux.org/current/bootstrap/x86_64-repodata' ...                            
[*] Updating repository `https://repo-default.voidlinux.org/current/x86_64-repodata' ...                                      
[*] Updating repository `https://repo-default.voidlinux.org/current/nonfree/x86_64-repodata' ...                              
[*] Updating repository `https://repo-default.voidlinux.org/current/debug/x86_64-repodata' ...                                
[*] Updating repository `https://repo-default.voidlinux.org/current/multilib/bootstrap/x86_64-repodata' ...                   
[*] Updating repository `https://repo-default.voidlinux.org/current/multilib/x86_64-repodata' ...                             
[*] Updating repository `https://repo-default.voidlinux.org/current/multilib/nonfree/x86_64-repodata' ...
=> xbps-src: updating software in / masterdir...
=> xbps-src: cleaning up / masterdir...
=> python3-pylast-5.3.0_1: removing autodeps, please wait...
=> python3-pylast-5.3.0_1: building with [python3-pep517] [python3] for x86_64...
   [host] hatchling-1.22.3_1: found (https://repo-default.voidlinux.org/current)
   [host] hatch-vcs-0.4.0_1: found (https://repo-default.voidlinux.org/current)
   [host] python3-build-1.2.1_1: found (https://repo-default.voidlinux.org/current)
   [host] python3-installer-0.7.0_2: found (https://repo-default.voidlinux.org/current)
   [check] python3-pytest-7.4.3_1: found (https://repo-default.voidlinux.org/current)
   [check] python3-flaky-3.8.1_1: found (https://repo-default.voidlinux.org/current)
   [check] python3-httpx-0.27.0_1: found (https://repo-default.voidlinux.org/current)
   [runtime] python3-httpx-0.27.0_1: found (https://repo-default.voidlinux.org/current)
=> python3-pylast-5.3.0_1: installing host dependencies: hatchling-1.22.3_1 hatch-vcs-0.4.0_1 python3-build-1.2.1_1 python3-installer-0.7.0_2 python3-pytest-7.4.3_1 python3-flaky-3.8.1_1 python3-httpx-0.27.0_1 ...
=> python3-pylast-5.3.0_1: running do_check ...
============================= test session starts ==============================
platform linux -- Python 3.12.4, pytest-7.4.3, pluggy-1.3.0
rootdir: /builddir/python3-pylast-5.3.0
configfile: pytest.ini
plugins: anyio-4.3.0, flaky-3.8.1
collected 141 items

tests/test_album.py sssssssssss                                          [  7%]
tests/test_artist.py sssssssssssssssssssssss                             [ 24%]
tests/test_country.py ss                                                 [ 25%]
tests/test_library.py sssss                                              [ 29%]
tests/test_librefm.py ss                                                 [ 30%]
tests/test_network.py ssssssssssssssssssssssssssssss                     [ 51%]
tests/test_tag.py ssss                                                   [ 54%]
tests/test_track.py sssssssssssssssssss                                  [ 68%]
tests/test_user.py ssssssssssssssssssssssssssssssssssss                  [ 93%]
tests/unicode_test.py .........                                          [100%]

======================== 9 passed, 132 skipped in 0.27s ========================
```

</p>
</details>

Reply:
Glad that CI is happy with it as honestly I don't know how to write checks in template.